docs/developer-guide/nodejs-api.md

```diff
@@ -153,23 +153,27 @@ var CLIEngine = require("eslint").CLIEngine;
 
 The `CLIEngine` is a constructor, and you can create a new instance by passing in the options you want to use. The available options are:
 
+* `baseConfig` - Set to false to disable use of base config. Could be set to an object to override default base config as well.
```

Comment:
Missing `allowInlineConfig`, `cacheFile` in this list (according to what is visible in the cli-engine.js changes below).